config/checker/com.intellij.astub

```diff
@@ -70,12 +70,22 @@ import org.checkerframework.checker.initialization.qual.UnknownInitialization;
 @UIPackage package com.intellij.openapi;
 
 
-package com.intellij.openapi.actionSystem;
+@UIPackage package com.intellij.openapi.actionSystem;
 
-class ToggleAction {
-  @UIEffect void setSelected(AnActionEvent e, boolean state);
+class AnAction {
+  @SafeEffect AnAction();
+  // We can safely mark `actionPerformed` and `update` as `@UIEffect` (the default given we're in a `@UIPackage`)
+  // since they are indeed invoked on the dispatch thread.
```

Comment:
Or at least so does `Application#isDispatchThread` say when I checked for our actions ;)